components/customer-dashboard.tsx

```diff
@@ -64,8 +68,34 @@ const CustomerDashboard = () => {
     enabled: !!customerStore?.customer?.id,
   });
 
+  // console.log(customerPointsResponse, "customerPointsResponse");
   const { mutate: sendTransaction, isPending, isError } = useSendTransaction();
 
+  const fetchOnChainPointsForAllTokens = async () => {
```

Comment:
I believe there is another PR for fetching the customer's on-chain points. Then why is this part of this PR as well?